cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -1,6 +1,5 @@
 (ns cljfmt.core-test
-  (:require [#?@(:clj (clojure.test :refer)
-                 :cljs (cljs.test :refer-macros)) [deftest testing is]]
+  (:require [clojure.test :refer [deftest testing is]]
```

Comment:
I can revert this change if you want, it slipped in, but it's no longer necessary to load `cljs.test` the way it was done prior to this PR.